include/termcolor/termcolor.hpp

```diff
@@ -9,9 +9,17 @@
 //! :license: BSD, see LICENSE for details
 //!
 
+#pragma once
+
 #ifndef TERMCOLOR_HPP_
 #define TERMCOLOR_HPP_
 
+#ifdef _MSC_VER
+#pragma warning(push)
+#pragma warning(disable: 4800) //about force cast int to bool
+#pragma warning(disable: 4005) //about changing macro in minwindef.h(130) (wtf?)
```

Comment:
I'd prefer to avoid any *WTF*s in the source code. Could you please help me to understand which line causes that warning?